sdk/src/main/scala/io/horizen/SidechainSettings.scala

```diff
@@ -26,8 +26,9 @@ case class WebSocketClientSettings(
 
 case class WebSocketServerSettings(
     wsServer: Boolean = false,
-    wsServerPort: Int = 8025
-) extends SensitiveStringer
+    wsServerPort: Int = 8025,
+    wsServerAllowedOrigins: Seq[String] = Seq()
```

Comment:
since the allowed list is implemented only in the account app, 
let's raise an exception if this property is detected and not empty  inside SidechainApp (utxo one)